Detectors/TPC/reconstruction/src/KrBoxClusterFinder.cxx

```diff
@@ -112,7 +86,53 @@ void KrBoxClusterFinder::fillADCValue(int cru, int rowInSector, int padInRow, in
     adcValue /= correctionFactor;
   }
 
-  mMapOfAllDigits[timeBin][rowInSector][corPad] = adcValue;
+  mTempTimeSlice[rowInSector][corPad] = adcValue;
+}
+
+void KrBoxClusterFinder::fillTempTimeSlice(std::vector<o2::tpc::Digit>& eventSector, const int timeSlice) {
```

Comment:
I don't get the point, that this removes digits which can not be used afterwards. Is this not just a copy of digits?
The other points are clear and I implemented them. Thank you.

Reply:
If we use the code in DPL we will receive the digits in the device. Since the function received the vector by reference, the vector will be empty at the end, since `loopOverSector` will loop over all time bins and `eventSector.erase(digit);` will remove the digit. So the `std::vector<o2::tpc::Digit>` passed from the DPL device to `loopOverSector` will then by empty in the DPL device.